src/main/resources/openapi/token-1.0.yaml

```diff
@@ -107,6 +106,40 @@ paths:
           $ref: "#/components/responses/trait_400"
         "500":
           $ref: "#/components/responses/trait_500"
+  /token/logout:
+    parameters:
+      - $ref: headers/okapi-tenant-required.yaml
+      - $ref: headers/okapi-url-required.yaml
+    post:
+      description: Logout a single token
+      operationId: token-logout
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/refreshToken"
```

Comment:
Who will copy the refresh token from the cookie to the body?
The front-end cannot do this because the front-end cannot read the refresh cookie.

Reply:
Mod login is calling this, not the front end. Mod-login has code for parsing the cookie and making the request to mod-login.